templates/docs/examples/patterns/tooltips/detached.html

```diff
@@ -0,0 +1,16 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Tooltips / Detached{% endblock %}
+
+{% block standalone_css %}patterns_tooltips{% endblock %}
+
+{% block content %}
+<button class="js-tooltip" aria-describedby="default-tooltip">
```

Comment:
the "js-tooltip" class doesn't seem to be needed

Reply:
oh yes, good point